targets/TARGET_STM/USBPhy_STM32.cpp

```diff
@@ -420,11 +438,18 @@ void USBPhyHw::remote_wakeup()
 const usb_ep_table_t *USBPhyHw::endpoint_table()
 {
     static const usb_ep_table_t table = {
+#if (MBED_CONF_TARGET_USB_SPEED != USE_USB_OTG_HS)
         1280, // 1.25K for endpoint buffers but space is allocated up front
+#else
+        4096,
+#endif
         {
             {USB_EP_ATTR_ALLOW_CTRL                         | USB_EP_ATTR_DIR_IN_AND_OUT, 0, 0},
             {USB_EP_ATTR_ALLOW_BULK | USB_EP_ATTR_ALLOW_INT | USB_EP_ATTR_DIR_IN_AND_OUT, 0, 0}, // NON ISO
             {USB_EP_ATTR_ALLOW_BULK | USB_EP_ATTR_ALLOW_INT | USB_EP_ATTR_DIR_IN_AND_OUT, 0, 0}, // NON ISO
+#if (MBED_CONF_TARGET_USB_SPEED == USE_USB_OTG_HS)
+            {USB_EP_ATTR_ALLOW_ALL                          | USB_EP_ATTR_DIR_IN_AND_OUT, 0, 0},
+#endif
```

Comment:
Maybe this change can be the default configuration for all?
and then remove #if (MBED_CONF_TARGET_USB_SPEED == USE_USB_OTG_HS) ?

Reply:
Maybe, but I don't have enough hardware variety to test it :slightly_smiling_face: